packages/prices/src/prices.ts

```diff
@@ -0,0 +1,31 @@
+import { PriceSpecification } from "@js-ligo/vocab";
+
+export interface DynamicPrice {
+  [key: string]: number;
+}
+
+export class Prices {
+  /**
+   * Find Total Price Static
+   */
+  async findTotalStaticPrice(givenPrice: PriceSpecification) {
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    const a = new Date(givenPrice.validThrough!);
```

Comment:
Does this mean we will only support `PriceSpecification`s that have a `validThrough` and `validFrom`? Or can we support prices that have these missing?

Reply:
> 

If `validThrough` and `validFrom` are missing, we would need something to represent total days the item would be used and priced for. Does this represent the validity of offer or the total time the item is being used?

Reply:
> Does this mean we will only support `PriceSpecification`s that have a `validThrough` and `validFrom`? Or can we support prices that have these missing?

We could create a different function for `odometer` based pricing by miles/kilometers

Reply:
It seems like `validFrom` and `validThrough` would be required when calculating price that is determined by days. But if the price is based on something else, like distance, these fields would not be needed.

This function should take an input of `PriceSpecification[]` and maybe other inputs needed to calculate the price. Like the `pickupTime` and `dropoffTime` values from a `RentalCarReservation` and maybe an optional `startOdometer` and `endOdometer`.

Reply:
Or perhaps a separate function for `odometer` like you mentioned.